syntax = "proto3";

package protobuf_gameplay;

message gamePlay {
  string gameName = 1;
}